### PR TITLE
RSS Feed (part 3)

### PR DESCRIPTION
**Pull request by rchs819**
- fix RSS feed thumbnail preview
- added handler for all 404 page
- added `abbreviateNumber` to `Number.prototype.abbr` for shortcut
- make items in `/static` folder cacheable for 1 year

### CODE REVIEW

**Comment by rchs819** [comment]
@roy7 Thanks.
Could you quickly run `npm install` to get required packages on the server? 
it looks like one of the module is missing.

**Comment by roy7** [comment]
Done. Any better?

**Comment by roy7** [comment]
Results were:

> npm WARN leela_zero_server@1.0.0 license should be a valid SPDX license expression
> npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.1.3 (node_modules/fsevents):
> npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.1.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

**Comment by rchs819** [comment]
Thanks. All good now

File: server.js [hunk 1]
```diff
@@ -986,32 +990,69 @@ app.post('/submit', (req, res) => {
...
+    if(!req.params.tab || req.params.tab == "matches") {
+        pug_data.matches = await db.collection("matches")
+            .aggregate([ 
+                { "$match": { $or : [{ network1 : network.hash }, { network2 : network.hash }] } },
+                { "$lookup": { "localField": "network2", "from": "networks", "foreignField": "hash", "as": "network2" } }, { "$unwind": "$network2" }, 
+                { "$lookup": { "localField": "network1", "from": "networks", "foreignField": "hash", "as": "network1" } }, { "$unwind": "$network1" }, 
+                { "$sort": { _id: -1 } }, 
+                { "$limit": 100 } 
+            ]).toArray();
+        console.log(pug_data.matches);
```
Comment (roy7) [inline server.js]: Is this a stray debugging line we can remove?

Comment (rchs819) [reply server.js]: Oh yes please remove. My bad.

**Pull request merged**